conf/include/webkit-bleeding.inc

```diff
@@ -3,11 +3,11 @@
 #
 #   require conf/include/webkit-bleeding.inc
 
-PREFERRED_VERSION_cog = "0.16+httpsAUTOINC+%"
+PREFERRED_VERSION_cog = "0.17+httpsAUTOINC+%"
```

Comment:
There is not a cog recipe available. 

You probably want to update that recipe too and probably adpat also the packageconfig like this https://github.com/Igalia/meta-wpe-image/commit/6d227876e69636e25063bad2d04014ecacddbf80